package-testing/dotnet-sdk-relay/EppoSDKRelay/Program.cs

```diff
@@ -14,5 +14,8 @@ public static IHostBuilder CreateHostBuilder(string[] args) =>
             {
                 builder.UseStartup<Startup>();
                 builder.UseUrls("http://" + host + ":" + port);
+
+                // Listen on the docker hostname as well
+                builder.UseUrls("http://host.docker.internal:" + port);

```

Comment:
May be accessed through the environment `SDK_RELAY_HOST` and `host.docker.internal` hostnames